build.gradle

```diff
@@ -128,5 +128,5 @@ codenarcTest {
 }
 
 jacoco {
-  toolVersion = "0.8.3"
+  toolVersion = "0.8.6"
```

Comment:
I think it would be nice to keep these out of the PR here, and move it to a separate PR, as the changes look completely unrelated ...

Reply:
Sure, I had issues with Java 11 when using the old version on my local machine. Will put this into a different PR.

Reply:
Resolved via https://github.com/opendevstack/ods-jenkins-shared-library/pull/388/commits/b1f7ec500e6e6f3d41bcd91be7f3d9140c922262